.github/Workflows/my-ci-pipeline.yml

```diff
@@ -0,0 +1,17 @@
+name: Continuous Integration
```

Comment:
I can't see any evidence that these tests are actually running in GitHub! I _think_ that's probably because the folder they're in is `Workflows` when GitHub expects `workflows`
E.g. there are no runs listed against this PR or [showing under Actions](https://github.com/abhilash-jagityla/DevOps-Course-Starter/actions)